libr/core/disasm.c

```diff
@@ -4610,16 +4610,17 @@ static char *ds_sub_jumps(RDisasmState *ds, char *str) {
 		name = fcn->name;
 	} else if (f) {
 		flag = r_core_flag_get_by_spaces (f, addr);
-		if (flag) {
-			name = flag->name;
-		} else {
-			RBinReloc *rel = getreloc (ds->core, addr, ds->analop.size);
+		RBinReloc *rel = getreloc (ds->core, addr, ds->analop.size);
```

Comment:
why resolve the reloc if we have a flag already? this is querying twice. and imho flags should go before the bin info

Reply:
@radare  the main purpose of this PR is to use Rbin info first then only try to resolve using flags because currently for every call to same reloc function there is a flag created but with a number appended and it looks bad in disasm 

![flag](https://user-images.githubusercontent.com/18589720/55160467-6a760680-5189-11e9-9dbf-e6b640841b70.png)


![ir](https://user-images.githubusercontent.com/18589720/55160473-6cd86080-5189-11e9-9812-afb9cb051b95.png)

But yeah i have fixed querying twice issue

Reply:
Can't we change the flags creation so that they don't contain the number suffix? As previously said, it should be possible to configure flagsitems to have a "displayed name" and a "real name" (that is unique), right?

Reply:
Yup that could be done xD ,  I just wanted quick fix :) 

I won't have time until weekend,  but yeah let me try that way np :)